capellambse/aird/parser/_box_factories.py

```diff
@@ -290,10 +290,12 @@ def part_factory(seb: C.SemanticElementBuilder) -> aird.Box:
         seb.styleclass = abstract_obj.attrib[C.ATT_XST].split(":")[-1]
         assert seb.styleclass is not None
         if seb.styleclass.endswith("Component"):
+            nature = ("Behavior", "Node")[abstract_obj.get("nature") == "NODE"]
             seb.styleclass = "".join(
                 (
                     seb.styleclass[: -len("Component")],
                     "Human" * (abstract_obj.get("human") == "true"),
+                    nature * (bool(abstract_obj.get("nature"))),
```

Comment:
This looks very weird to me. You're setting `nature` to one of two values depending on an XML attribute, but then here you're double-checking whether that attribute was set in the first place? Just set the variable to the correct value from the start. Keeps the logic together and avoids having to double-check.